### PR TITLE
[Demo] Updated config for farmer usecase

### PR DESCRIPTION
**Pull request by jainhitesh9998**

### CODE REVIEW

File: farmer-credential.vm [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#set( $calenderNow = $date.getCalendar() )
+#set( $ISO8601DateTimeWithMillisUTC = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'" )
+## Adding no. of Days we want to set as expiry, here '5' represent Day and '30' represent no. of days
+#set( $expiry =$calenderNow.add(5, 30))
+#set( $formattedDateOfBirth = $date.toDate("yyyy-MM-dd", ${dateOfBirth}))
+
+{
+"@context": [
+"https://www.w3.org/2018/credentials/v1",
+"https://example.com/farmer-profile-context.json"
```
Comment (vijay151096) [inline farmer-credential.vm:12]: can we replace with proper context file.

File: farmer-credential.vm [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+],
+"type": [
+"VerifiableCredential",
+"FarmerCredential"
```
Comment (vijay151096) [inline farmer-credential.vm:16]: As per confluence, shouldn't it be "FarmerProfileCredential"

Comment (jainhitesh9998) [reply farmer-credential.vm:16]: updated it to farmercredential in all places, added it for consistency

File: certify-sunbird-farmer.properties [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                            'type': {'display': {{'name': 'Farming Types','locale': 'en'}}},\
+                            'landArea': {'display': {{'name': 'Land Area','locale': 'en'}}},\
+                            'landOwnershipType': {'display': {{'name': 'Land Ownership Type','locale': 'en'}}},\
+                            'primaryCropType': {'display': {{'name': 'Primary Crop Type','locale': 'en'}}},\
+                            'secondaryCropType': {'display': {{'name': 'Secondary Crop Type','locale': 'en'}}}\
+                     }},\
+                    'display': {{'name': 'Farmer', \
+                                  'locale': 'en', \
+                                  'logo': {'url': 'https://${mosip.api.public.host}/inji/veridonia-logo.png','alt_text': 'a square logo of a Veridonia'},\
+                                  'background_color': '#FDFAF9',\
```
Comment (vijay151096) [inline certify-sunbird-farmer.properties]: can you please use the farmer theme colour.

File: certify-sunbird-farmer.properties [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+                                  'background_image': { 'uri': 'https://${mosip.api.public.host}/inji/veridonia-logo.png' }, \
```
Comment (vijay151096) [inline certify-sunbird-farmer.properties]: we have to use the agristack logo
https://raw.githubusercontent.com/aranaravi/kendo-ui-core/master/Agro%20Vertias%20Logo.svg

File: certify-sunbird-farmer.properties [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+                            'works': {'display': {{'name': 'Work Type','locale': 'en'}}},\
+                            'type': {'display': {{'name': 'Farming Types','locale': 'en'}}},\
+                            'landArea': {'display': {{'name': 'Land Area','locale': 'en'}}},\
+                            'landOwnershipType': {'display': {{'name': 'Land Ownership Type','locale': 'en'}}},\
+                            'primaryCropType': {'display': {{'name': 'Primary Crop Type','locale': 'en'}}},\
+                            'secondaryCropType': {'display': {{'name': 'Secondary Crop Type','locale': 'en'}}}\
+                     }},\
+                    'display': {{'name': 'Farmer', \
+                                  'locale': 'en', \
+                                  'logo': {'url': 'https://${mosip.api.public.host}/inji/veridonia-logo.png','alt_text': 'a square logo of a Veridonia'},\
```
Comment (vijay151096) [inline certify-sunbird-farmer.properties]: use https://raw.githubusercontent.com/aranaravi/kendo-ui-core/master/Agro%20Vertias%20Logo.svg

File: certify-sunbird-farmer.properties (hunk 4 above)
Comment (vijay151096) [reply certify-sunbird-farmer.properties]: please update for all version drafts

File: certify-sunbird-farmer.properties [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+              'display': {{'name': 'Farmer Credential', 'locale': 'en'}},\
+              'credential_configurations_supported' : { \
+                 'FarmerCredential' : {\
+                    'format': 'ldp_vc',\
+                    'scope' : 'sunbird_rc_insurance_vc_ldp',\
+                    'cryptographic_binding_methods_supported': {'did:jwk'},\
+                    'credential_signing_alg_values_supported': {'Ed25519Signature2020'},\
+                    'proof_types_supported': {'jwt': {'proof_signing_alg_values_supported': {'RS256', 'PS256'}}},\
+                    'credential_definition': {\
+                      'type': {'VerifiableCredential','FarmerCredential'},\
```
Comment (vijay151096) [inline certify-sunbird-farmer.properties]: Shouldn't it be "FarmerProfileCredential"

File: certify-sunbird-farmer.properties [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+## ------------------------------------------- Plugin enable properties ------------------------------------------------------------
+mosip.certify.integration.scan-base-package=io.mosip.certify.sunbirdrc.integration
+mosip.certify.integration.vci-plugin=SunbirdRCVCIssuancePlugin
+mosip.certify.integration.audit-plugin=LoggerAuditService
+
+## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
+mosip.certify.vciplugin.sunbird-rc.enable-psut-based-registry-search=false
+mosip.certify.vciplugin.sunbird-rc.issue-credential-url=${mosip.sunbird.url}/credentials-service/credentials/issue
+mosip.certify.vciplugin.sunbird-rc.supported-credential-types=FarmerCredential
```
Comment (vijay151096) [inline certify-sunbird-farmer.properties]: Shouldn't it be "FarmerProfileCredential" as per confluence
https://mosip.atlassian.net/wiki/spaces/Inji/pages/1426784342/Farmer+Credential+-+Use+Case

**Review by vijay151096 — APPROVED** [review]
lgtm

**Pull request merged**